### PR TITLE
Emily dev

### PR DESCRIPTION
**Pull request by emily-tsai11**
Add script to write cutflow tables in LaTeX format. Non-trivial changes to helper/cutflow.py and README.md. Output of cutflow.py matches summary plots. Other trivial changes are just whitespace changes for readability. 

The tau dz histogram (and all previous commits) are included here because I didn't make a new branch for each commit.

### CODE REVIEW

File: helper/cutflow.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        for reg in regions:
+            Counts[yr][f][reg] = {}
+            total = 0.0
+            for c in charges:
+                Counts[yr][f][reg][c] = {}
+                for ch in channels:
+                    h = file.Get(c + '_' + ch + '_' + reg + '_llM')
+                    nbinsX = h.GetXaxis().GetNbins()
+                    err = ROOT.Double()
+                    integral = h.IntegralAndError(1, nbinsX, err)
```
[the remark this answers is not in the document]
Comment (Jingyan95) [reply helper/cutflow.py]: The comment above is outdated

**Review by Jingyan95 — COMMENTED** [review]
I have some minor comments on cutflow.py

File: helper/cutflow.py
[the remark this answers is not in the document]
Comment (emily-tsai11) [reply helper/cutflow.py:1]: Done!

[the remark this answers is not in the document]
Comment (Jingyan95) [reply helper/cutflow.py:1]: > Done!

looks good! I like the design

**Pull request merged**